sys/src/9/amd64/devacpi.c

```diff
@@ -1860,7 +1860,7 @@ static void acpiintr(Ureg *u, void *v)
 	//if (sts & 1)
 	//	print("acpiintr: power button\n");
 	// XXX serve other interrupts here.
-	setpm1sts(sts);
+	setpm1sts(v && 1 || 0);
```

Comment:
This is ... cryptic. I don't understand what it is doing?

Reply:
`setpm1sts` is expecting an int, see https://github.com/Harvey-OS/harvey/blob/main/sys/src/9/amd64/devacpi.c#L1791
That is being passed on to `setbanked`, see https://github.com/Harvey-OS/harvey/blob/main/sys/src/9/amd64/devacpi.c#L1754
and that determines whether to print the verbose `"setbanked: ...` messages (https://github.com/Harvey-OS/harvey/blob/main/sys/src/9/amd64/devacpi.c#L1757) which are flooding the console (#977).

Reply:
Um, I guess, since `v` is used in a different sense in `setbanked`, we should give some more verbose (pun intended) var names; this patch here is actually causing another issue. Woops.

Reply:
Different patch coming in :sweat_smile: